src/extension.ts

```diff
@@ -44,7 +44,7 @@ export async function activate(context: ExtensionContext) {
 }
 
 async function isEmberCliProject(): Promise<boolean> {
-  const emberCliBuildFile = await workspace.findFiles('ember-cli-build.js')
+  const emberCliBuildFile = await workspace.findFiles('**/ember-cli-build.js', '**/{dist,tmp,node_modules}/**', 100)
```

Comment:
I guess since we only care about find *at least* one file we could also set `maxResults` to 1 instead?

not a blocker though, this already seems better than what we had before.